test-data/channels/dummy-optional-dependencies/noarch/repodata.json

```diff
@@ -1,528 +1,55 @@
 {
-    "info": {
```

Comment:
Should we keep a few of these tests? Just to make sure the extras work as expected?

Reply:
OK, I saw that you have one functionality test. I guess that's fine.